components/eamxx/src/physics/p3/atmosphere_microphysics.cpp

```diff
@@ -275,8 +282,11 @@ void P3Microphysics::initialize_impl (const RunType /* run_type */)
   }
   diag_inputs.ni_activated    = get_field_in("ni_activated").get_view<const Pack**>();
   diag_inputs.inv_qc_relvar   = get_field_in("inv_qc_relvar").get_view<const Pack**>();
+
+//OG why is this not using p3_preproc?
+//OG do we want dry or wet pressure here?
   diag_inputs.pres            = get_field_in("p_dry_mid").get_view<const Pack**>();
```

Comment:
This has the same problem as the Fortran code I think; because `qv_sat` wants a wet pressure, passing a dry pressure here is a bit wrong, even though `dpres` *does* need to be dry.